app/views/user_mailer/reset_password_instructions.zh-cn.text.erb

```diff
@@ -1,7 +1,7 @@
 <%= @resource.email %> ，嗨呀!!
 
-有人（但愿是你）请求更改你 Mastodon 账户的密码。如果是你的话，请点击下面的链接：
+有人（但愿是你）请求更改你Mastodon账户的密码。如果是你的话，请点击下面的链接：
```

Comment:
I may revert this whitespace change, per W3C CLREQ &sect; 3.2.2 and personal preference.